### PR TITLE
Cc/bugfix/documentation form elements

### LINKED ISSUE #352: Form Elements Documentation
**Describe the bug**
I'm checking if all component's text is updated. Inside form elements the first 2 elements does not have text associated (label, question, textarea, option bar, handling errors). Please close it if it is not an issue. 

https://designsystem.wmnetwork.co.uk/components/form-elements/

### LINKED ISSUE #353: Update form elements documentation (dev)
**Describe the bug**
Documentation needs to be updated for the following components:
- checkboxes
- dropdown
- input
- radios

Possible some new texts from here #352

### PR DESCRIPTION
**Pull request by catiarodriguescosta**
Delete Option bar component (single commit).
All documentation is updated / completed.

Fixes #352 
Fixes #353

### CODE REVIEW

File: src/www/pages/components/form-elements/index.njk [hunk 1]
```diff
@@ -72,20 +104,21 @@
...
   <ul>
-    <li>When user needs to enter short amount of text, e.g. name, email.</li>
+    <li>When user needs to enter short amount of text, e.g. name, email. </li>
   </ul>
   <h3>When not to use it?</h3>
   <ul>
     <li>When user needs to add a lot of text that might go over multiple lines</li>
     <li>Use fixed width inputs for content that has known length, e.g. postcode.</li>
   </ul>
+  <br>
```
Comment (gldgrnt) [inline src/www/pages/components/form-elements/index.njk:121]: You can remove this so that the spacing is consistent

**Review by gldgrnt — CHANGES_REQUESTED** [review]
All good except for a couple of tiny things

File: src/www/pages/components/form-elements/index.njk [hunk 2]
```diff
@@ -4,7 +4,19 @@
...
+  <h3>What does it do?</h3>
+  <ul><li>Tells users what a field is for.</li></ul>
+  <h3>When to use it?</h3>
+  <ul>
+    <li>When you are requesting multiple pieces of information to answer the same question. For example, the day, month and year of a date.</li>
+    <li>The label must appear directly above the field.</li>
+  </ul>
+  <h3>When not to use it?</h3>
+  <ul>
+    <li>o not use this component to ask more questions within a question - instead, ask a new question.</li>
```
Comment (daylesalmon) [inline src/www/pages/components/form-elements/index.njk:17]:
```suggestion
    <li>Do not use this component to ask more questions within a question - instead, ask a new question.</li>
```

**Review by daylesalmon — CHANGES_REQUESTED** [review]
Just fix the few changes @gldgrnt raised at it should be good to go.

I can see you have added a tag when you removed the options bar component which is great!

File: src/www/pages/components/form-elements/index.njk (hunk 1 above)
Comment (catiarodriguescosta) [reply src/www/pages/components/form-elements/index.njk]: sorted!

**Review by daylesalmon — APPROVED** [review]

**Review by daylesalmon — APPROVED** [review]

**Pull request merged**